step.yml

```diff
@@ -37,6 +37,9 @@ type_tags:
 - test
 is_always_run: false
 is_skippable: false
+deps:
+  brew:
+  - name: xcbeautify
```

Comment:
Is this a good idea? This will slow down every step run, even if the log formatter is not set to `xcbeautify`.

Reply:
And if there is a breaking change in `xcbeautify` package in the future, it will instantly break our step, for everyone.

Reply:
Discussed to remove dep, as it is preinstalled, and we do not want to check if installed in all cases (only when xcbeautify is selected). If it is not installed, Step will fall back to xcodebuild and print an error message.